### DISCUSSION TITLE
A small question about vue with scss style

### DISCUSSION THREAD

**Discussion by Tomcxf** [FranckFreiburger/vue3-sfc-loader#191] [Q&A]
Hi developer!
Thanks for your vue3-sfc-loader! It really help me a lot !
I am a new for vue and I plan to insert a vue component directly to html.
With the example in main page, I have successfully add vue component into html.
However, since some styles are written in scss, it doesn't show well.
I have seen that your publish - #181
But i‘m little confused about it.
1. How should I combine it with example code
2. since the scss written directly in my .vue like
```
<template>
...
</template>
<script>
...
</script>
<style lang="scss" scoped>
.brain-wrap {
  position: relative;
  z-index: 2;
  width: 1000px;
  transform: scale(0.7);
...
```
by the way, it doesn't import any other files
so how could i modify your example code.
I tried to replace 
```
<style lang="sass">
@import './_variables.scss';
body { background-color: $myColor }
</style>
```
to
```
<style lang="scss" scoped>
.brain-wrap {
  position: relative;
  z-index: 2;
  width: 1000px;
  transform: scale(0.7);
....
``` 
but it doesn't work.
How should I change to make scss work! Thanks!

**Comment by FranckFreiburger** [comment]
Hello, I don't quite understand your problem, then I made [the example](https://github.com/FranckFreiburger/vue3-sfc-loader/discussions/181) more complete ...hoping this helps you.
